fuzzer/src/main.rs

```diff
@@ -445,8 +457,6 @@ fn time_needed(sample: &str) -> Duration {
     // perform actual time measurement
     let clock = Clock::<ThreadCpuTime>::now();
     let parser = Parser::new_ext(sample, Options::all());
-    parser.for_each(|evt| {
-        black_box::black_box(evt);
-    });
+    black_box::black_box(parser.count());
```

Comment:
https://github.com/raphlinus/pulldown-cmark/pull/286#discussion_r280590293

Reply:
Forgot about that -- good catch!